modules/social_features/social_user_export/social_user_export.module

```diff
@@ -11,9 +11,23 @@
 function social_user_export_file_download($uri) {
   $scheme = \Drupal::service('file_system')->uriScheme($uri);
   $target = file_uri_target($uri);
-  $access = \Drupal::currentUser()->hasPermission('administer users');
 
-  if ($scheme === 'private' && preg_match('/^csv\/export-users-([a-f0-9]{12})\.csv$/i', $target) && $access) {
+  // Get the file to see who the owner is.
+  $query = \Drupal::entityQuery('file');
+  $query->condition('uri', $uri);
+  $fid = $query->execute();
+
+  /* @var \Drupal\file\FileInterface $file */
+  $file = \Drupal\file\Entity\File::load(reset($fid));
```

Comment:
```suggestion
  $file = File::load(reset($fid));
```
Don't forget to add the `use` statement at the top though.

Reply:
Done